crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -219,80 +220,237 @@ impl InterfaceApi for RealAdminInterfaceApi {
+    /// or an [`AdminResponse::Error`]
+    ///
+    /// [`AdminInterfaceConfig`]: ../config/struct.AdminInterfaceConfig.html
+    /// [`AdminResponse::AdminInterfacesAdded`]: enum.AdminResponse.html#variant.AdminInterfacesAdded
+    /// [`AdminResponse::Error`]: enum.AppResponse.html#variant.Error
     AddAdminInterfaces(Vec<AdminInterfaceConfig>),
-    /// Install an app from a list of Dna paths
-    /// Triggers genesis to be run on all cells and
-    /// Dnas to be stored
+    /// Install an app from a list of `Dna` paths.
+    /// Triggers genesis to be run on all `Cell`s and to be stored.
+    /// An `App` is intended for use by
+    /// one and only one Agent and for that reason it takes an `AgentPubKey` and
+    /// installs all the Dnas with that `AgentPubKey` forming new `Cell`s.
+    /// See [`InstallAppPayload`] for full details on the configuration.
+    ///
+    /// Note that the new `App` will not be "activated" automatically after installation
+    /// and can be activated by calling [`AdminRequest::ActivateApp`].
+    ///
+    /// Will be responded to with an [`AdminResponse::AppInstalled`]
+    /// or an [`AdminResponse::Error`]
+    ///
+    /// [`InstallAppPayload`]: ../../../holochain_types/app/struct.InstallAppPayload.html
+    /// [`AdminRequest::ActivateApp`]: enum.AdminRequest.html#variant.ActivateApp
+    /// [`AdminResponse::AppInstalled`]: enum.AdminResponse.html#variant.AppInstalled
+    /// [`AdminResponse::Error`]: enum.AppResponse.html#variant.Error
     InstallApp(Box<InstallAppPayload>),
-    /// List all installed [Dna]s
+    /// List the hashes of all installed `Dna`s.
+    /// Takes no arguments.
+    ///
+    /// Will be responded to with an [`AdminResponse::DnasListed`]
+    /// or an [`AdminResponse::Error`]
+    ///
+    /// [`AdminResponse::DnasListed`]: enum.AdminResponse.html#variant.DnasListed
+    /// [`AdminResponse::Error`]: enum.AppResponse.html#variant.Error
     ListDnas,
-    /// Generate a new AgentPubKey
+    /// Generate a new AgentPubKey.
+    /// Takes no arguments.
+    ///
+    /// Will be responded to with an [`AdminResponse::AgentPubKeyGenerated`]
+    /// or an [`AdminResponse::Error`]
+    ///
+    /// [`AdminResponse::AgentPubKeyGenerated`]: enum.AdminResponse.html#variant.AgentPubKeyGenerated
+    /// [`AdminResponse::Error`]: enum.AppResponse.html#variant.Error
     GenerateAgentPubKey,
-    /// List all the cell ids in the conductor
+    /// List all the cell ids in the conductor.
+    /// Takes no arguments.
+    ///
+    /// Will be responded to with an [`AdminResponse::CellIdsListed`]
+    /// or an [`AdminResponse::Error`]
+    ///
+    /// [`AdminResponse::CellIdsListed`]: enum.AdminResponse.html#variant.CellIdsListed
+    /// [`AdminResponse::Error`]: enum.AppResponse.html#variant.Error
     ListCellIds,
-    /// List all the active app ids in the conductor
+    /// List the ids of all the active (activated) `App`s in the conductor.
```

Comment:
Nitpick: The backticks make it seem to me like "App" is an actual data structure or otherwise reified concept, but it's not. It's an abstract concept with various structures and processes associated with it. I don't feel too strongly about this but just feels more right to have:

```suggestion
    /// List the ids of all the active (activated) Apps in the conductor.
```

in other words:

```
s/`App`/App/g
```